### PR TITLE
Refactor xml export descriptions

### LINKED ISSUE #155: More data in the by user RSS feed
Hi,

I've recently discovered the _by user_ RSS feed and this is really a great tool, providing very fast feedback on errors in latest changesets, making it easy to keep up to date with Osmose errors!

However, the RSS entries are somehow minimalistic and in my opnion, they could use a bit more details. What about providing the same infos as in the error infobox in the RSS entries?

At the moment, I often have to click the item (actually, the "E" link as the main link on the title does not auto-open the tooltip for the right marker) to have the details (summary of the error, etc).

Additionally, having the "mark fixed" and "discard as false positive" links right in the RSS entry would be super useful as well to handle Osmose issues quicker.

Thanks!
Best,

### PR DESCRIPTION
**Pull request by gileri**
Fixes #155, #339

### CODE REVIEW

**Comment by frodrigo** [comment]
Argh. No. I just move out from template.

I plan to drop template support and even bottle.

**Comment by gileri** [comment]
So what approach would you suggest ? Making HTML templates by concatenating strings is not really enjoyable nor readable.

**Comment by frodrigo** [comment]
Maybe using lxml again and dump it as string into the xml.

**Comment by gileri** [comment]
Is the template OK before I rewrite it with lxml ? I'd like to avoid rewriting it thrice ;)

**Comment by frodrigo** [comment]
Content look ok for me. Be the layout have to be tested in readers.

**Comment by gileri** [comment]
lxml and string concatenations are now used render the XML descriptions.
The feed is also now valid according to the [W3C feed validator](https://validator.w3.org/feed/).

Before :
Before

After :
After

File: web_api/issues.py [hunk 1]
```diff
@@ -201,31 +202,70 @@ def xml_issue(res, website, lang, query, main_website, remote_url_read):
...
+                    H.A(_("Mark issue as fixed"), href="http://{}/api/0.3/issue/{}/done".format(website, res["uuid"])),
+                    " ",
+                    H.A(_("Mark issue as false positive"), href="http://{}/api/0.3/issue/{}/false".format(website, res["uuid"])),
+                )
+            )
+            plain_desc += "\nOsmose: {}".format(map_url)
+            if "tags" in e:
+                html_desc.append(H.P("Tags:"))
+                tags_ul = H.UL()
+                plain_desc += "\nTags:"
```
Comment (frodrigo) [inline web_api/issues.py:242]: "Tags" can be translated

**Comment by frodrigo** [comment]
@jocelynj what do think about the .pot update ?

File: web_api/issues.py (hunk 1 above)
Comment (gileri) [reply web_api/issues.py:242]: Done, nice catch.

**Review by gileri — COMMENTED** [review]
I added wrapped the "Tags:" string for translation and updated `osmose-frontend.pot` once again.

**Comment by jocelynj** [comment]
> @jocelynj what do think about the .pot update ?

The change seems fine. Could somebody check that running "make" keep the changes?

File: web_api/issues.py (hunk 1 above)
Comment (frodrigo) [reply web_api/issues.py:242]: Can also translate the plain text.

**Comment by gileri** [comment]
>Could somebody check that running "make" keep the changes?

If that helps that's what I did to obtain those changes.

**Comment by frodrigo** [comment]
Thank you.

Deployed.

**Comment by jocelynj** [comment]
@gileri : I'm getting an issue when launching make here. Do I need to run npm before launching it?
```
% make
find: ../../web_api/views: No such file or directory
```

**Comment by frodrigo** [comment]
This PR does not change how the .pot is generated. On the the .pot file.

**Pull request merged**